_pages/calculator.md

```diff
@@ -9,67 +9,57 @@ sitemap: true
 ---
 
 <div class="calculator-block">
-	<p>
-		Si hubieras invertido 
-		<input id="invest-quantity" type="number" value="1000">
-		<select id="invest-fiat">
-			<option>EUR</option>
-			<option>USD</option>
-		</select> en 
-
-		<span class="normal-selector">
+    <div class="calculator-form-row">
+        <div class="calculator-col-start">
+            <label>Inversión</label>
+        </div>
+        <div class="calculator-col-end">
+            <input id="invest-quantity" type="number" value="1000" autofocus>
+            <select id="invest-fiat">
+                <option>EUR</option>
+                <option>USD</option>
+            </select>
+        </div>
+    </div>
+    <div class="calculator-form-row">
+        <div class="calculator-col-start">
+            <label>Criptomoneda</label>
+        </div>
+        <div class="calculator-col-end">
 			<select id="invest-currency" onchange="updateInputMinDate()">
 				<option value="BTC"  min="2010-07-18">Bitcoin</option>
 				<option value="ETH"  min="2015-08-08">Ethereum</option>
 				<option value="LTC"  min="2013-09-15">Litecoin</option>
 				<option value="XMR"  min="2015-01-27">Monero</option>
 				<option value="DASH" min="2014-02-04">Dash</option>
 				<option value="XRP"  min="2015-01-30">Ripple</option>
-				<option class="editable">otro</option>
+				<option class="editable">Otra moneda...</option>
 			</select>
-			<input class="editOption" autofocus/>
```

Comment:
why remove the autofocus?

Reply:
Why do we have "autofocus" in a hidden form box?
I moved it to the first visible inpux box